node/src/neighborhood/mod.rs

```diff
@@ -5857,9 +5867,9 @@ mod tests {
             new_ip: new_public_ip,
         });
 
-        assert_eq!(
+        assert_ne!(
```

Comment:
Can you put in a comment here to explain why the assertion is weak?

"Sometimes this test runs against the small test `dbip_country.rs`, and sometimes it runs against the big generated `dbip_country.rs` with real data; this assertion must succeed in both cases."

Or something...